common/utils/src/main/java/org/apache/spark/internal/LoggerFactory.java

```diff
@@ -19,6 +19,11 @@
 
 public class LoggerFactory {
 
+  public static Logger getLogger(String name) {
```

Comment:
`YarnShuffleService` will use it:
<img width="711" alt="image" src="https://github.com/apache/spark/assets/15246973/8a4e79bd-43e2-4995-9e0a-57a047bd1e50">